src/drawing/area.rs

```diff
@@ -106,6 +106,7 @@ pub struct DrawingArea<DB: DrawingBackend, CT: CoordTranslate> {
     backend: Rc<RefCell<DB>>,
     rect: Rect,
     coord: CT,
+    inset: bool,
```

Comment:
Like I said, it's a little bit coupling between the drawing and the chart context here.

Reply:
Looks good for me, but you are master. Do as you wish.